client/jetpack-cloud/sections/partner-portal/assign-license-form/style.scss

```diff
@@ -29,17 +30,68 @@
 
 .assign-license-form__top {
 	display: flex;
+	flex-wrap: nowrap;
 	justify-content: space-between;
-}
+	align-items: center;
+	margin-bottom: 1rem;
 
-.assign-license-form__top > div {
-	align-self: flex-end;
-	margin-bottom: 0;
+	@include breakpoint-deprecated( '>660px' ) {
+		flex-wrap: wrap;
+	}
+
+	@include break-medium {
+		flex-wrap: nowrap;
+	}
 }
 
-.assign-license-form__top {
-	margin-bottom: 1rem;
-	margin-top: -1.5rem;
+.assign-license-form__controls {
+	display: flex;
+	flex-direction: column-reverse;
+	align-items: center;
+	flex-grow: 1;
+	position: fixed;
```

Comment:
[We've just merged another PR where we moved the control to the bottom](https://github.com/Automattic/wp-calypso/blob/81e56378e2b802af79d37b562586922b2565c8ae/client/jetpack-cloud/sections/partner-portal/issue-license-form/style.scss#L32) with 90% the same behaviour. Would it make sense to try and generalise the form controls?

Or perhaps add a task to generalise it to a shared form/header component?